### PR TITLE
⏰  1.12.0

### PR DESCRIPTION
**Pull request by nataliescottdavidson**
Release 1.12.0 adds support for cron triggers, quality of life improvements, bug fixes, and updates docs.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,99 @@
...
+
+  - **Update stalebot settings - [ispivey], [pull/1561]**
+
+    Stalebot now waits 180 days to mark stale, it marks 'timed out' instead of 'wontfix', and added 'never stale' tag.
+
+    [ispivey]: https://github.com/ispivey
+    [pull/1561]: https://github.com/cloudflare/wrangler/pull/1561
+
+  - **Pin Rust to 1.47 and fix clippy lints - [ObsidianMinor], [pull/1609]**
+    [obsidianminor]: https://github.com/ObsidianMinor
```
Comment (ashleymichal) [inline CHANGELOG.md:53]: need a newline between the heading and the link pin

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,7 +109,7 @@
 
     When running `wrangler dev` as an authenticated user, your requests will now run on the same servers that Cloudflare Workers run on in production. This means that what you see is what you get. `wrangler dev` should behave exactly like production, though we still recommend deploying to a staging website before going to production in order to ensure your changes are safe. This change means you get access to things like `request.cf`, the Cache API, and any Cloudflare settings you've applied in the dashboard while developing your Workers.
 
-    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [everlastingbugstopper]: https://github.com/EverlastingBugstopper
```
Comment (ashleymichal) [inline CHANGELOG.md:112]: idk why all of these got downcased, but they should not be. check your editor settings? it looks like a similar thing happened with `*` -> `_` and some whitespace corrections.

**Review by ashleymichal — CHANGES_REQUESTED** [review]
just some formatting snafus

**Review by ObsidianMinor — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,113 @@
...
+    [pull/1532]: https://github.com/cloudflare/wrangler/pull/1532
+    [pull/1526]: https://github.com/cloudflare/wrangler/pull/1526
+    [pull/1519]: https://github.com/cloudflare/wrangler/pull/1519
+
+  - **Docs for ARM users - [rathboma], [pull/1499]**
+
+    [rathboma]: https://github.com/rathboma
+    [pull/1499]: https://github.com/cloudflare/wrangler/pull/1499
+
+# Changelog
```
Comment (ashleymichal) [inline CHANGELOG.md:109]: remove redundant changelog heading.

**Review by ashleymichal — APPROVED** [review]
just one v. smol change: we don't need `# Changelog` twice

**Pull request merged**